test/tables.jl

```diff
@@ -137,7 +137,7 @@ Base.propertynames(d::DuplicateNamesColumnTable) = (:a, :a, :b)
         cat = CategoricalVector(["hey", "there", "sailor"])
         cat2 = [c for c in cat] # Vector of CategoricalString
         nt = (a=cat, b=cat2)
-        df = DataFrame(nt)
+        df = DataFrame(nt, copycolumns=false)
```

Comment:
Maybe also test when `copycolumns=true`?

Reply:
I will add more tests covering all cases (hopefully) when we finalize the the functionality.

Reply:
I have added more tests, but this one is waiting for @quinnj to comment on general Tables.jl (and DataStreams.jl) design.